lib/puppet-syntax/manifests.rb

```diff
@@ -15,7 +15,7 @@ def check(filelist)
       end
 
       # Catch syntax warnings.
-      Puppet::Util::Log.newdestination(Puppet::Test::LogCollector.new(errors))
+      Puppet::Util::Log.newdestination(Puppet::Test::LogCollector.new(output))
       Puppet::Util::Log.level = :warning
```

Comment:
Ah that makes sense, I wasn't sure. Either way I think it'd be good to use the log levels that Puppet supplies if possible - I'm working on some code now to test if that's feasible.